#Header header    # ROS Message headaer

int32 command   # Command ID
uint8 axis      # Axis number
float32 fval    # Float value 
uint32 ival     # int value

